packages/angular_devkit/build_angular/src/builders/browser/schema.json

```diff
@@ -29,15 +29,71 @@
       "type": "array",
       "default": [],
       "items": {
-        "$ref": "#/definitions/extraEntryPoint"
+        "oneOf": [
+          {
+            "type": "object",
+            "properties": {
+              "input": {
+                "type": "string",
+                "description": "The file to include.",
+                "pattern": "\\.[cm]?[j]sx?$"
```

Comment:
Is this supposed to be `\\.[cm]?[jt]sx?$`? Does it allow TypeScript? If not, could we simplify `[j]` to just `j`?

Same point in a couple other places in the PR.

Reply:
No, TypeScript files are not allowed. Updated the regexp to remove the square braces.